Eastern-Research-Group/csb-rebate-forms-app
# Feature/add initial 2024 prf support

## Related Issues:
* CSBAPP-488

## Main Changes:
* Add initial support for 2024 PRF.

NOTE: We're waiting on the BAP team's 2024 FRF ETL/orchestrator work to complete, before we can test these changes. As soon as that's done and the BAP is processing 2024 FRF submissions, we can test perform the testing steps below.

## Steps To Test:
1. Navigate to the dashboard.
2. Ensure at least one of your 2024 FRF submissions' status is set to "Accepted" in the BAP.
3. Click the "New Payment Request" button.
4. Ensure a new 2024 PRF submission has been created, and you're redirected to the page to continue with filling in the form.
5. Fill in the form fields, submit it, and ensure you're taken back to the dashboard.
